uw-frame-components/portal/main/partials/username.html

```diff
@@ -15,14 +15,23 @@
     <span>{{sessionCtrl.firstLetter}}</span>
     <md-tooltip class="top-bar-tooltip" md-direction="bottom" md-delay="500">Options</md-tooltip>
   </md-button>
-  <md-menu-content width="3">
-    <md-menu-item ng-if="$storage.showSettings">
+  <md-menu-content width="5" class="top-bar-menu-content username-menu-content">
+    <md-menu-item class="md-2-line">
+      <div class="md-list-item-text">
+        <p class="display-name">{{ sessionCtrl.username }}</p>
+        <p class="campus-id" ng-if="sessionCtrl.campusId != ''">Campus ID: {{ sessionCtrl.campusId }}</p>
+      </div>
+      <a ng-href="/profile/emergencyInfo" ng-click="pushGAEvent('Top Bar', 'Click Link', 'Profile');" aria-label="see your profile information" class="link__see-all md-secondary">
```

Comment:
How can we avoid hard-coding the URL? Do we have a way to check whether a /profile app exists in a given environment, or should we show this only to UW-Madison users (who we *know* has a /profile)?